### ISSUE TITLE
dodiscovery does not handle lldp failures in some cases

### ISSUE THREAD

**Issue by russauld** [xcat2/xcat-core#5360]
The code checks for `"$myswitch" != "Agent instance for device not found"` but some Dell switches return extra info at the end of the error message.
Some examples are:
```
Agent instance for device not found u300000005
Agent instance for device not found u300000006
Agent instance for device not found u300000004
```

The check should be a regexp match instead.

**Comment by chuckbrazie** [comment]
I am out of the office until 07/16/2018.

I am out of the office until  Monday July 16.


Note: This is an automated response to your message  "Re: [xcat2/xcat-core]
dodiscovery does not handle lldp failures in some cases (#5360)" sent on
7/2/18 9:25:18 PM.

This is the only notification you will receive while this person is away.

**Comment by chuckbrazie** [comment]
I am out of the office until 07/16/2018.

I am out of the office until  Monday July 16.


Note: This is an automated response to your message  "[xcat2/xcat-core]
dodiscovery does not handle lldp failures in some cases (#5360)" sent on
7/2/18 8:57:49 PM.

This is the only notification you will receive while this person is away.

**Comment by immarvin** [comment]
hi @zet809 , can you take a look at this?

**Comment by chuckbrazie** [comment]
I am out of the office until 07/16/2018.

I am out of the office until  Monday July 16.


Note: This is an automated response to your message  "Re: [xcat2/xcat-core]
dodiscovery does not handle lldp failures in some cases (#5360)" sent on
7/3/18 1:29:27 AM.

This is the only notification you will receive while this person is away.

**Comment by zet809** [comment]
This fix will be included in 2.14.3 release. So close this issue.
